test/extended/networking/internal_ports.go

```diff
@@ -130,7 +130,7 @@ var _ = ginkgo.Describe("[sig-network] Internal connectivity", func() {
 							protocol := protocols[k]
 							testingMsg := fmt.Sprintf("[%s: %s -> %s:%d]", protocol, from.Spec.NodeName, to.Spec.NodeName, ports[k])
 							testMsg := fmt.Sprintf("%s-from-%s-to-%s", "hello", from.Status.PodIP, to.Status.PodIP)
-							command, err := testRemoteConnectivityCommand(protocol, "localhost:"+strconv.Itoa(nodeTCPPort), to.Spec.NodeName, ports[k], testMsg)
+							command, err := testRemoteConnectivityCommand(protocol, "localhost:"+strconv.Itoa(nodeTCPPort), to.Status.HostIP, ports[k], testMsg)
```

Comment:
is there a possibility that the status doesn't have `hostip`?

Reply:
no

Reply:
is the node IP available elsewhere?

Reply:
been there since 1.17 : https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/